rand_chacha/src/chacha.rs

```diff
@@ -61,11 +61,21 @@ impl<T> fmt::Debug for Array64<T> {
         write!(f, "Array64 {{}}")
     }
 }
+impl<T> ::core::cmp::PartialEq for Array64<T>
```

Comment:
Some places used full paths for the ops traits, so I wasn't sure if that's preferred here.

Reply:
I don't care a lot. Since Edition 2018 you don't need the leading `::`.